### PR TITLE
Mod mark

### LINKED ISSUE #79: Implement Mod Mark
Marks module(s) of a batchmate as taken, which means the batchmate has taken the module(s) before.

### PR DESCRIPTION
**Pull request by carriezhengjr**
Add `mod mark` command to allow the student to mark module(s) of a batchmate as taken. It will also check if all modules entered by student exist in the list of modules.

Tests for the command are not completed yet.

Fixes #79

### CODE REVIEW

File: src/main/java/seedu/address/logic/commands/ModCommand.java [hunk 1]
```diff
@@ -2,17 +2,19 @@
...
     // TODO: Add the other mod commands
-    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Modifies the modules related to a person. "
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Modifies the modules related to a batchmate. "
             + "\nParameters: INDEX (must be a positive integer) MODULE [MORE_MODULES]..."
             + "\nExamples:"
-            + "\nmod add INDEX MODULE [MORE_MODULES]... : Adds the entered mods to the person at the index"
-            + "\nmod delete INDEX MODULE [MORE_MODULES]... : Deletes the entered mods from the person at the index";
+            + "\nmod add INDEX MODULE [MORE_MODULES]... : Adds the entered mods to the batchmate at the index."
+            + "\nmod delete INDEX MODULE [MORE_MODULES]... : Deletes the entered mods from the batchmate at the index."
+            + "\nmod mark INDEX MODULE [MORE_MODULES]... : Marks the entered mods of the batchmate "
```
Comment (ElijahS67) [inline src/main/java/seedu/address/logic/commands/ModCommand.java:16]: Maybe there should be a full stop here? :sweat_smile:
```suggestion
            + "\nmod mark INDEX MODULE [MORE_MODULES]... : Marks the entered mods of the batchmate. "
```

**Review by ElijahS67 — APPROVED** [review]
Tested and working fine on my end, LGTM!

**Comment by sltsheryl** [comment]
Works for me too.

File: src/main/java/seedu/address/logic/commands/ModCommand.java (hunk 1 above)
Comment (carriezhengjr) [reply src/main/java/seedu/address/logic/commands/ModCommand.java:16]: Alright, added. Thanks! 😄

**Pull request merged**